Stratis.Bitcoin/Features/RPC/Controllers/FullNodeController.cs

```diff
@@ -110,6 +117,28 @@ public GetInfoModel GetInfo()
             return model;
         }
 
+        /// <summary>
+        /// Implements getblockheader RPC call
+        /// </summary>
+        /// <param name="hash">hash of block</param>
```

Comment:
Please use an upper case at the beginning of XML documentation phrases, and a dot in the end.